wandb/sdk/launch/_project_spec.py

```diff
@@ -39,9 +44,13 @@ def __init__(
     ):
 
         self.uri = uri
+        self.source_tag = uri.split("/")[-1]
+        self.build_image = False
```

Comment:
I've added this, in the case that we download the entrypoint, we always need to rebuild the image so we copy the new file into the image.